frontend/src/locales/de.json

```diff
@@ -26,8 +26,22 @@
     "community": "Gemeinschaft",
     "continue-to-registration": "Weiter zur Registrierung",
     "current-community": "Aktuelle Gemeinschaft",
+    "myContributions": "Meine Beiträge",
     "other-communities": "Weitere Gemeinschaften",
-    "switch-to-this-community": "zu dieser Gemeinschaft wechseln"
+    "switch-to-this-community": "zu dieser Gemeinschaft wechseln",
+    "writing": "Schreiben"
+  },
+  "contribution": {
+    "activity": "Tätigkeit",
+    "formText": {
+      "h3": "Dein Beitrag zum Gemeinwohl",
+      "lastMonth": "Für <b>{month} {year}</b> kannst du noch <b>{creation}</b> GDD einreichen.",
+      "text1": "Bring dich mit deinen Talenten in die Gemeinschaft ein! Dein freiwilliges Engagement honorieren wir mit 20 GDD pro Stunde bis maximal 1.000 GDD im Monat.",
+      "text2": "Beschreibe deine Gemeinwohl-Tätigkeit mit Angabe der Stunden und trage einen Betrag von 20 GDD pro Stunde ein! Nach Bestätigung durch einen Moderator wird der Betrag deinem Konto gutgeschrieben.",
+      "thisMonth": "Für <b>{month} {year}</b> kannst du noch <b>{creation}</b> GDD einreichen. "
```

Comment:
What is the difference to `lastMonth`

Reply:
lastMonth is the full date of new Date() - 1 month, this is the sentence that is then placed on the form.  Here I only need the month written out as text.